### PR TITLE
Responsive design, fixed access levels on degrees

### PR DESCRIPTION
**Pull request by christykobyletska**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by andrii52 — APPROVED** [review]

File: src/pages/userPage/ActiveMembership/ActiveMembership.tsx [hunk 1]
```diff
@@ -103,7 +103,9 @@ const ActiveMembership = () => {
 
 
   const IsUserHasAccessToManageDegree = (userRoles: Array<string>): boolean => {
-    return (userRoles?.includes(Roles.CityHead) && currentUser.cityId==user.cityId) ||
+    return (userRoles?.includes(Roles.KurinHead) && currentUser.kurinId==user.kurinId) ||
```
Comment (danylo-melnychuk) [inline src/pages/userPage/ActiveMembership/ActiveMembership.tsx:106]:
```suggestion
    return (userRoles?.includes(Roles.KurinHead) && currentUser.kurinId == user.kurinId) ||
```

File: src/pages/userPage/ActiveMembership/ActiveMembership.tsx [hunk 2]
```diff
@@ -302,10 +304,10 @@ const ActiveMembership = () => {
                   </div>
                 </React.Fragment>
               ))}
-              {IsUserHasAccessToManageDegree(roles?.map((role:any)=>{
-                      if(!(role === Roles.KurinHead || role === Roles.KurinHeadDeputy ))
+               {IsUserHasAccessToManageDegree(roles?.map((role:any)=>{
```
Comment (danylo-melnychuk) [inline src/pages/userPage/ActiveMembership/ActiveMembership.tsx:307]:
```suggestion
               {IsUserHasAccessToManageDegree(roles?.map((role:any) => {
             
```

Comment (christykobyletska) [reply src/pages/userPage/ActiveMembership/ActiveMembership.tsx]: done

File: src/pages/userPage/ActiveMembership/ActiveMembership.tsx (hunk 1 above)
Comment (christykobyletska) [reply src/pages/userPage/ActiveMembership/ActiveMembership.tsx]: done

**Review by OleksandrBilan — APPROVED** [review]

**Review by nicoolle — APPROVED** [review]

**Pull request merged**